modules/smtp/site.js

```diff
@@ -427,3 +436,264 @@ function smtpSettingsPageHandler() {
         );
     });
 }
+$(function () {
```

Comment:
master has passed over these changes. If you have additional lines of code, ensure they are in the right route handlers.